install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -103,3 +103,67 @@ spec:
             severity: warning
           annotations:
             message: "Memory usage of {{ $value | humanize }} on {{ $labels.node }} exceeds 90%. Master nodes starved of memory could result in degraded performance of the control plane."
+    - name: control-plane-cpu-utilization
+      rules:
+        - alert: HighOverallControlPlaneCPU
+          annotations:
+            summary: >-
+              CPU utilization across all three control plane nodes is higher than two control plane nodes can sustain; a single control plane node outage may
+              cause a cascading failure; increase available CPU.
+            runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneCPU.md
+            description: >-
+              Given three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
+              This is because if a single control plane node fails, the remaining two must handle the load of the cluster in order to be HA.
```

Comment:
nit: probably worth spelling out "highly available".  I expect most readers will already have that acronym internalized, but it doesn't take much longer to read spelled out, and there may be some readers who have not internalized the acronym.